docker/search.dev.json

```diff
@@ -0,0 +1,10 @@
+{
+  "GATEWAY_URL": "http://unstable--dummy-api.libero.pub",
```

Comment:
agreed in person to postpone running a local `dummy-api`, since this is not used by tests (only local development)